mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
+1 on private `_pick_data_channels(exclude='bads')` that just wraps to `pick_types(info, meg=True, eeg=True, seeg=True, exclude=exclude)`. That way if we add a new channel type of some sort, we only change it in one place (DRY).